openassessment/fileupload/backends/base.py

```diff
@@ -30,7 +30,9 @@ class Settings(object):
         'image/jpeg': '.jpg',
         'image/pjpeg': '.jpg',
         'image/png': '.png',
-        'application/pdf': '.pdf'
+        'application/pdf': '.pdf',
+        'application/msword': '.doc',
+        'application/vnd.openxmlformats-officedocument.wordprocessingml.document': '.docx'
```

Comment:
Looks good, thanks!  Next time, include a trailing comma on the last line so that future diffs are nice and clean.

Reply:
Sure. thanks @nedbat